lua/sps/cl_init.lua

```diff
@@ -15,14 +15,50 @@ end)
 
 concommand.Add("ps_togglemenu", function(ply, cmd, args) PS_ToggleMenu() end)
 
+PS_DarkMode = true
+
+concommand.Add("ps_toggletheme", function(ply, cmd, args)
+	if PS_DarkMode then
```

Comment:
Can you factor this out to a function like SetPointshopTheme(bool dark), and then call that function so the defaults are set in the same place so there is no duplicate configuration?

Reply:
Do you mean something like this?
`
function SetPointshopTheme(dark)
    PS_DarkMode = dark
end
SetPointshopTheme(true)
`

Reply:
That's without saved convars, it would probably be something like `SetPointshopTheme(convar:GetBool())` once I put that in

Reply:
nevermind, I figured it out